tajo-catalog/tajo-catalog-server/src/main/java/org/apache/tajo/catalog/dictionary/InfoSchemaMetadataDictionary.java

```diff
@@ -91,7 +91,7 @@ public String getSystemDatabaseName() {
     return systemTableNames;
   }
   
-  private TableDescriptor getTableDescriptor(String tableName) {
+  private TableDescriptor getTableDescriptor(String tableName) throws UndefinedTableException {
```

Comment:
I know you just modified the function signature to throw an exception. However, I found a problem that this function returns null instead of throwing UndefinedTableException when the table of the given name is not found.
Would you also check it please?